llama_stack/providers/inline/agents/meta_reference/agent_instance.py

```diff
@@ -222,13 +227,23 @@ async def create_and_execute_turn(self, request: AgentTurnCreateRequest) -> Asyn
             )
             await self.storage.add_turn_to_session(request.session_id, turn)
 
-            chunk = AgentTurnResponseStreamChunk(
-                event=AgentTurnResponseEvent(
-                    payload=AgentTurnResponseTurnCompletePayload(
-                        turn=turn,
+            if output_message.tool_calls:
```

Comment:
can we add a parameter to the `create_turn` which gates this behavior? we can make that parameter be default False, so older clients on v0.1.3 will continue to function identically and then our new client code can pass that param to be True for this release. Then in v0.1.5, we can completely remove that parameter. that will retain backward compatibility.

Reply:
Good idea! Added `allow_turn_resume` flag, and works perfectly with client v0.1.3. re-syncing client